packages/commonwealth/client/scripts/views/pages/view_thread/ViewThreadPage.tsx

```diff
@@ -52,6 +52,7 @@ import { LinkedThreadsCard } from './linked_threads_card';
 import { LockMessage } from './lock_message';
 import { ThreadPollCard, ThreadPollEditorCard } from './poll_cards';
 import { SnapshotCreationCard } from './snapshot_creation_card';
+import { LinkedUrlCard } from './linked_url_card';
```

Comment:
linked_url_card => LinkedUrlCard

Reply:
used this as it looked like the standard for all of the other cards
<img width="267" alt="image" src="https://github.com/hicommonwealth/commonwealth/assets/36428666/8a7d7f7c-f584-437c-8e66-f88b3110738c">

Reply:
fixed per doc